duckduckgo/iOS
# only show global stats after visiting at least 10 sites

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/482056236481737
CC:

**Description**:

Currently the global stats are shown as soon as there's three networks to do display. This change adds the constraint of visiting more than 10 sites.

**Steps to test this PR**:

1. go to duckduckgo.com, open privacy protection and note the still gathering data message 
1. Visit 10 sites (including DDG that'll be 11) and check the global stats still shows the gathering data message until the 11th site at which point it shows the Top 3
1. The user can still see the leaderboard at any time by tapping in that area
